frodo.js

```diff
@@ -1,35 +1,34 @@
-const truncatedString = string.substring(0, maxLength) + (string.length > maxLength ? "..." : "");
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?